### DISCUSSION TITLE
Is it possible to use DP-ZBL architecture in deepmd-kit ?

### DISCUSSION THREAD

**Discussion by y1u0d2a1i** [deepmodeling/deepmd-kit#1932] [Q&A] [answered]
Hello,
I read your paper called 'Deep learning inter-atomic potential model for accurate irradiation damage simulations'. 
I'd like to use DP-ZBL architecture for sputtering simulation, but it was not mentioned in the documentation. So, I want to ask, is it possible to use DP-ZBL architecture in deepmd-kit? or just use hybrid potential in Lammps?

**Comment by Chengqian-Zhang** [comment]
Hi , DP-ZBL architecture is supported in deepmd-kit now . As for the parameters , I will give you a example .

**Reply by Chengqian-Zhang** [reply]
![image](https://user-images.githubusercontent.com/100290172/191162099-1b9b4ab3-5c1b-4820-aa01-9b0bf4b36d3f.png)

**Reply by y1u0d2a1i** [reply]
Thank you for your reply. I'll try it !!

**Reply by Chengqian-Zhang** [reply]
You are welcome, feel free to tell me if you have any questions

**Comment by Chengqian-Zhang** [comment, marked as answer]
https://docs.deepmodeling.com/projects/deepmd/en/master/train/train-input.html?highlight=srtab
The meaning of the parameters is described in this document.
1、"use_srtab" : The table for the short-range pairwise interaction added on top of DP . This table needs to be prepared by the user . 
2、"smin_alpha" : The short-range tabulated interaction will be swithed according to the distance of the nearest neighbor. This distance is calculated by softmin. This parameter is the decaying parameter in the softmin.  In general,  "smin_alpha" takes 0.1.
3、"sw_rmin" : The lower boundary of the interpolation between short-range tabulated interaction and DP
4、"sw_max" : The upper boundary of the interpolation between short-range tabulated interaction and DP
